### ISSUE TITLE
Create Water pump

### ISSUE THREAD

**Issue by Beezone32** [TeamAOF/All-of-Fabric-6#311]
Create water pumps are no longer pulling from water pipes for a water source , along with when pulled up in ponder will crash the game

**Comment by Linguardium** [comment]
please post the crashlog

**Comment by WElfz** [comment]
My Create water pump setup suddenly stop to work here too after updating to the new version of the pack. But, nothing of game crashes here

**Comment by Beezone32** [comment]
-
[crash-2023-07-15_13.00.34-client.txt](https://github.com/TeamAOF/All-of-Fabric-6/files/12062225/crash-2023-07-15_13.00.34-client.txt)
Here is the crash report

**Comment by Linguardium** [comment]
might be an issue with Roots Classic adding a newer version of porting_lib than the older version of Create uses.

**Comment by Linguardium** [comment]
The crashing on the 2nd section of the 2nd page of the pipe ponder (where it shows pulling fluid) is a Create bug. it happens with only Create and Fabric API present but is fixed in the lastest create version. 

The being unable to pull from a pipe is caused by an updated Porting Lib from roots, which breaks Create's ability to pump from source blocks at all....

**Comment by WORMSTweaker** [comment]
Can confirm that they are indeed broken now, can't Roots Classic be downgraded to a previous version where porting_lib was the same as Create's one? Or is there no newer version of Create available maybe?

**Comment by Linguardium** [comment]
pack dev is waiting on 1 more create addon to update, then Create can be updated to resolve this issue.

**Comment by Linguardium** [comment]
This should be resolved by the latest pack update. Can you confirm?

**Comment by WORMSTweaker** [comment]
Yes, last updates fixed it, confirmed fixed since Create was updated
